src/diff/index.js

```diff
@@ -294,9 +294,12 @@ function diffElementNodes(
 		if (newVNode.type === null) {
 			return document.createTextNode(newProps);
 		}
+
+		const builtIn = newProps.is;
+
 		dom = isSvg
 			? document.createElementNS('http://www.w3.org/2000/svg', newVNode.type)
-			: document.createElement(newVNode.type);
+			: document.createElement(newVNode.type, builtIn && { is: builtIn });
```

Comment:
Super awesome! Played around with it a bit and we can save 8 bytes by inlining the variable declaration. This presumably works because we're using `newProps` a lot already, so it's already available in the gzip directory:

```js
document.createElement(newVNode.type, newProps.is && { is: newProps.is });
```

Reply:
Superb! Thanks Marvin